### PR TITLE
fix bad google drive link in pet tutorial

### PR DESCRIPTION
**Pull request by bendhouseart**
closes issue #384

### CODE REVIEW

**Review by Remi-Gau — APPROVED** [review]
looks good to me

File: src/tutorials/pet.md [hunk 1]
```diff
@@ -19,13 +19,13 @@ and others use proprietary formats for example ECAT format (.v).
 In order to facilitate easy testing of data conversion across different PET file formats
 the [OpenNeuroPET project][OpenNeuroPET project] has compiled a bunch of phantom data
 from different scanner types and is distributing two examples
-[here](https://drive.google.com/file/d/10S0H7HAnMmxHNpZLlifR14ykIuiXcBAD/view?usp=sharing).
```
Comment (Remi-Gau) [inline src/tutorials/pet.md:22]: Can I let you fix them on the bids website too? (the tutorials have been migrated)

https://github.com/bids-standard/bids-website/blob/522d8fb5288cb569d5c2013f505e1c2ab5c73a23/docs/getting_started/tutorials/conversion/pet.md?plain=1#L22

**Comment by Remi-Gau** [comment]
Do you think it'd be possible to put this data on OSF or somewhere else where the half life of the link would be better?

File: src/tutorials/pet.md (hunk 1 above)
Comment (bendhouseart) [reply src/tutorials/pet.md:22]: Yes, this is where I got into trouble fixing this earlier; making an update-broken-links PR on the bids-website to main branch too.

**Comment by bendhouseart** [comment]
> Do you think it'd be possible to put this data on OSF or somewhere else where the half life of the link would be better?

I was planning on putting it into an s3 bucket, but I can do OSF instead.

**Pull request merged**